dockstore-integration-testing/src/test/java/io/dockstore/client/cli/CheckerWorkflowIT.java

```diff
@@ -94,7 +95,11 @@ public void testCWLToolAddCheckerRefreshPublishUnpublish() throws ApiException {
         DockstoreTool githubTool = containersApi.registerManual(newTool);
 
         // Refresh the workflow
-        containersApi.refresh(githubTool.getId());
+        DockstoreTool refresh = containersApi.refresh(githubTool.getId());
```

Comment:
It's a good start. 
Let's get input files tested as well. 
Also let's get both tested for workflows as well as tools.